src/sidebar/reducers/links.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Reducer for storing a "links" object in the Redux state store.
+ *
+ * The links object is initially null, and can only be updated by completely
+ * replacing it with a new links object.
+ *
+ * Used by serviceUrl.
+ */
+
+'use strict';
+
+/** Return the initial links. */
+function init() { return {links: null}; }
+
+/** Return updated links based on the given current state and action object. */
+function update(state, action) { return {links: action.newLinks}; }
+
+/** Return an action object for updating the links to the given newLinks. */
+function action(newLinks) {
+  return { type: 'UPDATE_LINKS', newLinks: newLinks };
+}
```

Comment:
The world's simplest reducer. By the way it's easy to generalise this kind of reducer (holds a single object, and updating just means completely replacing the object with a new one) so that the links reducer could be something like: `var links = SingleValueReducer('links');`